paper/user_study.tex

```diff
@@ -23,29 +19,35 @@ \subsection{Research Questions}\label{subsec:research_questions}
 
 \subsection{Methodology}\label{subsec:methodology} % Minye
 
-\paragraph{Potential Users}
+Our user study is qualitative and consists of five interviews.
+We do not perform any major quantitative analysis of the results as the sample size is too small to draw any statistically significant conclusions.
+
+\subsubsection{Potential Users}
 We look for potential users to conduct the interviews.
-There are two basic standards we follow to find a potential user:
+We consider the following groups of people as potential users:
 \begin{itemize}
     \item Professors and students who are interested in our application.
-    \item People who frequently use \cfgfiles and schema languages
+    \item People who frequently use \cfgfiles{} and schema languages
 \end{itemize}
+In table~\ref{tab:user_study_feedback}, the professions of the participants are listed.
 
-\paragraph{Interview Questions}
-For the interview, we created a JSON schema and \cfgfile{}, about a made-up self-driving car simulation,
+\subsubsection{Interview Questions}
+We created a JSON schema and \cfgfile{}, about a made-up self-driving car simulation.
```

Comment:
```suggestion
We created a JSON schema and \cfgfile{} for a made-up self-driving car simulation software.
```
@beyond2void like that maybe?

Reply:
Oh sorry I mean line 36: "The interview questions deal with working with those files."

Reply:
True